standalone/runtime/tests/test_multisig.rs

```diff
@@ -64,3 +67,61 @@ fn integration_test_multisig() {
         assert_eq!(CollateralCurrency::total_balance(&account_of(EVE)), 20_000_000_000_001);
     });
 }
+
+#[test]
+fn integration_test_multisig_vesting() {
+    ExtBuilder::build().execute_with(|| {
+        let vesting_amount = 30_000_000;
+        let multisig_account = MultiSigPallet::multi_account_id(&vec![account_of(ALICE), account_of(BOB)], 2);
+
+        // vested transfer takes free balance of caller
+        assert_ok!(Call::Tokens(TokensCall::set_balance {
+            who: multisig_account.clone(),
+            currency_id: INTR,
+            new_free: vesting_amount,
+            new_reserved: 0,
+        })
+        .dispatch(root()));
+
+        // gradually release amount over 100 periods
+        let call = Call::Vesting(VestingCall::vested_transfer {
+            dest: account_of(EVE),
+            schedule: VestingSchedule {
+                start: 0,
```

Comment:
This will probably be in the past actually, the crowdloan start height. It just means that user's can claim more initially based on the current height but it should work as expected.